src/backend/src/services/teams.services.ts

```diff
@@ -162,4 +162,45 @@ export default class TeamsService {
 
     return teamTransformer(updateTeam);
   }
+
+  static async setTeamLead(submitter: User, teamId: string, userId: number): Promise<Team> {
+    const team = await prisma.team.findUnique({
+      where: { teamId },
+      ...teamQueryArgs
+    });
+
+    const newLead = await prisma.user.findUnique({
+      where: { userId }
+    });
+
+    if (!team) throw new NotFoundException('Team', teamId);
+    if (!newLead) throw new NotFoundException('User', userId);
+
+    if (!isAdmin(submitter.role) && submitter.userId !== team.headId) {
+      throw new AccessDeniedException('You must be an admin or the head to update the lead!');
+    }
+
+    if (newLead.userId === team.headId || team.members.includes(newLead)) {
```

Comment:
there should be a function for this in shared called like makeTeamList aslso these will all be lists not just one person